src/index.html

```diff
@@ -3,11 +3,735 @@
+          <a href="#header" class="help__content-utilis-link">Where to go for a warranty</a>
+          <a href="#header" class="help__content-utilis-link">Service Policy</a>
+        </div>
+        <div class="help__content-contact">
+          <a href="tel:+86-0571-86105373" class="help__content-contact-link">
+            +86-0571-86105373
+          </a>
+
+          <a href="mailto:global@katvr.com"  class="help__content-contact-link">
+            global@katvr.com
+          </a>
+
+          <a
+          href="mailto:overseas@katvr.com" class="help__content-contact-link">
+            overseas@katvr.com
+          </a>
+        </div>
+      </div>
+    </section>
+
+    <main class="main">
+      <div class="background-wrapper background-wrapper--more-than-gaming" id="benefits">
+        <section class="more-than-gaming">
+          <h2 class="more-than-gaming__title">
+            MORE THAN <span class="more-than-gaming__title--color">GAMING!</span>
+          </h2>
+          <p class="more-than-gaming__paragraph-customized">
+            This also made for people who are interested in...
+          </p>
+
+          <article class="more-than-gaming__item">
+            <a href="#">
+              <img
+              src="images/slogan-education.svg"
+              alt="student biretta icon"
+              class="more-than-gaming__item-image"
+              >
+            </a>
+            <h3 class="more-than-gaming__item-subtitle more-than-gaming__item-subtitle--education">EDUCATION</h3>
+            <p class="more-than-gaming__item-paragraph">
+              Create aducational simulations, trainings and much more with...
+            </p>
+            <p class="more-than-gaming__item-paragraph-alternative">
+              Create aducational simulations, trainings and much more with unlimited virtual space and minimum physical space
+            </p>
+          </article>
+
+          <article class="more-than-gaming__item">
+            <a href="#">
+              <img
+              src="images/slogan-real-estate.svg"
+              alt="biret-icon"
+              class="more-than-gaming__item-image"
+              >
+            </a>
+            <h3 class="more-than-gaming__item-subtitle more-than-gaming__item-subtitle--real-estate">REAL ESTATE</h3>
+            <p class="more-than-gaming__item-paragraph">
+              Desighn architectural projects in a deeply realistic environment...
+            </p>
+            <p class="more-than-gaming__item-paragraph-alternative">
+              Desighn architectural projects in a deeply realistic environment allowing visitors to freely walk around, and feel their vibe
+            </p>
+          </article>
+
+          <article class="more-than-gaming__item">
+            <a href="#">
+              <img
+              src="images/slogan-fitness.svg"
+              alt="biret-icon"
+              class="more-than-gaming__item-image"
+              >
```

Comment:
Need to fix it everywhere
```suggestion
              <img
                src="images/slogan-fitness.svg"
                alt="biret-icon"
                class="more-than-gaming__item-image"
              >
```